src/pybind/mgr/dashboard/controllers/docs.py

```diff
@@ -99,33 +212,49 @@ def _gen_responses_descriptions(cls, method):
             resp['202'] = {'description': "Operation is still executing."
                                           " Please check the task queue."}
 
+        if resp_object:
+            for status_code, response_body in resp_object.items():
+                resp[status_code].update({
```

Comment:
you need to cast `status_code` to string `str(status_code)`

Reply:
This is already handled in __init__.py and it works for me. Do you get any errors?

Reply:
I had some errors but it was probably my fault. It's working now.